en/docs/tutorials/scenarios/scenario9-realtime-data.md

```diff
@@ -1,4 +1,44 @@
 # Scenario 9 - Realtime Data with WebSocket API
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Quantis wants to provide real time location of their trains to their customers. The sensors in the train will 
+be providing real time events and Quantis wants to convert it to websockets and expose it as a Streaming API 
+so that the client mobile applications can subscribe to it and receive real time events.**
```

Comment:
```suggestion
so that the client mobile applications can subscribe to it and receive real time events.
```
Let's avoid bolding this section as that affects readability. Please apply everywhere.